src/Client.js

```diff
@@ -108,20 +108,28 @@ export default class Client extends EventTarget {
 
      this.webSocket = new WebSocketEndpoint(this);
 
+     /**
+      * The logged in client account. Set in [login]{@link Client#login} and
+      * unset in [logout]{@link Client#logout}.
+      * @type {Account | undefined}
+      */
+     this.account = undefined;
+
+
     for (let event of ["message", "messageupdate", "messagedelete", "autherror"]) {
       this.webSocket.addEventListener(event, evt => this.dispatchEvent(evt));
     }
   }
 
   /**
    * Sign up for a new account with a single initial alias.
-   * @param {string} alias The name of the initial alias if signup is successful.
+   * @param {string} aliasName The name of the initial alias if signup is successful.
    * @param {string} email The email address associated with the account.
    * @param {string} password The password associated with the account.
    * @returns {Promise<{message: string}>} A validation message
    */
-  signup(alias, email, password) {
-    return this.auth.signup(alias, email, password);
+  signup(aliasName, email, password) {
```

Comment:
not sure if we want to use "aliasName" everywhere or just "alias".

Reply:
Just alias (or username in this case, since it's the first one)